widgetssdk/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,7 @@
     <string name="glia_chat_attachment_remove_item_content_description">Remove attachment %1$s</string>
     <string name="glia_chat_visitor_status_transferring">Transferring</string>
     <string name="glia_chat_new_messages">New Messages</string>
+    <string name="glia_chat_gva_gallery_card_message_content_description">%1$s \n Card: %2$d of %3$d.</string>
```

Comment:
Is this synced with iOS? If so then it should be renamed based on the conventions established

Reply:
Not synced yet.